### DISCUSSION TITLE
[Web] Arrows not working properly

### DISCUSSION THREAD

**Discussion by aleripe** [singerdmx/flutter-quill#540] [Web]
My issue is about [Web] (I've not tested on other platforms yet)

I have tried running `example` directory successfully before creating an issue here. Upgrated to latest version of Flutter 2.8.1.

You can reproduce the issue by running the `example` with Chrome. Paste a long text like some Lorem Ipsum paragraphs. Then try going up and down with the arrows. Instead of going to the previous/next line, cursor jumps to an earlier/later position inside the same line. I think this is not the desired behavior.

**Comment by NicolasDionB** [comment]
I'm experiencing the same problem. Has anyone tested it on mobile or desktop? The jumps made using the arrow keys seems ratter erratic.

@singerdmx I think this should be treated as an issue though, it's clearly not the usual text editing behavior. I would fix it but I don't master the project yet! If you (or someone else!) have any insights on this, that could help!

**Reply by singerdmx** [reply]
The logic is probably in lib/src/widgets/keyboard_listener.dart

**Comment by NicolasDionB** [comment]
Just a quick update: I still have the issue using the version 7.3.0 (Flutter 3.10.6). See the example bellow, I'm just using the arrow up/down keys (running on Chrome / Windows 11):

![Text_edit_arrow_keys](https://github.com/singerdmx/flutter-quill/assets/25512495/84cf46a4-c971-4d9c-a7d8-c9511e48efff)

Coming back up using the up key I can tell it's not random but it's not really useful to the user as the position is not what is expected, sometimes I get line jumps as well.

I think this should be promoted as an issue instead of a discussion.
